### PR TITLE
run in docker info & logger/panic cleanup

### PR DESCRIPTION
**Pull request by swetharepakula**

### CODE REVIEW

**Review by MHBauer — APPROVED** [review]

File: cmd/broker/main.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ func main() {
 		logger.Fatal("main", err)
 	}
 
-	broker := broker.BlockheadBroker{}
+	broker := &broker.BlockheadBroker{}
```
Comment (nimakaviani) [inline cmd/broker/main.go]: why is this an issue? we only create that object once.

File: cmd/broker/main.go [hunk 2]
```diff
@@ -16,13 +17,13 @@ func main() {
...
 	if len(os.Args) < 2 {
-		panic("config file missing")
+		logger.Fatal("main", errors.New("config file is missing"))
 	}
 
 	configFilepath := os.Args[1]
 	cfg, err := config.NewConfig(configFilepath)
 	if err != nil {
-		panic(err)
+		logger.Fatal("main", err)
```
Comment (nimakaviani) [inline cmd/broker/main.go]: LGTM

File: README.md [hunk 3]
```diff
@@ -11,4 +11,4 @@ Run the broker by doing:
 
 OR
 
-    docker build -t blockhead . && docker run blockhead
+    docker build -t blockhead . && docker run -v $(pwd)/config.json:/config.json -p 3333:3333 blockhead
```
Comment (nimakaviani) [inline README.md]: LGTM

**Review by nimakaviani — DISMISSED** [review]

File: cmd/broker/main.go (hunk 1 above)
Comment (swetharepakula) [reply cmd/broker/main.go]: That is true that we only use it once. I didn't think about that. I do prefer the pointer receiver but  don't feel strongly about this either way.

Comment (nimakaviani) [reply cmd/broker/main.go]: I would say lets keep it as is. i dont see any added value with the pointer receiver. otherwise good to merge.

**Pull request merged**